playbooks/roles/rabbitmq/defaults/main.yml

```diff
@@ -17,9 +17,9 @@ RABBIT_USERS:
   - name: 'admin'
     password: "{{ RABBIT_ADMIN_PASSWORD }}"
   - name: 'edx'
-    password: 'edx'
+    password: "{{ XQUEUE_RABBITMQ_PASS }}"
   - name: 'celery'
-    password: 'celery'
+    password: "{{ EDXAPP_CELERY_PASSWORD }}"
```

Comment:
Sorry, my bad. In new installs this variable appears from my-passwords.yml when installing open edX with security recommendations https://openedx.atlassian.net/wiki/display/OpenOPS/How+to+Override+Default+Configuration+Passwords+and+Verify+Exposed+Services